support.md

```diff
@@ -69,15 +67,15 @@ In the description of your issue, be sure to also include:
     `systemctl status brickman.service -l`.
   * If a screenshot of the EV3 would be helpful, run `fbgrab <some-name>.png`
     on the EV3 and include the picture in your comments.
-<p />
```

Comment:
I'm afraid it is the latter :laughing: I am not particularly well-learned in this area, so I can't say how this is viewed in general use. I know that only _void elements_, which can never include content (such as `br`), can be self-closing in HTML5; however that might not have been the case for XHTML or HTML4.